av1an-core/src/progress_bar.rs

```diff
@@ -144,23 +144,23 @@ pub fn reset_mp_bar_at(pos: u64) {
   }
 }
 
-pub fn init_multi_progress_bar(len: u64, workers: usize) {
+pub fn init_multi_progress_bar(len: u64, workers: usize, total_chunks: usize) {
   MULTI_PROGRESS_BAR.get_or_init(|| {
     let mpb = MultiProgress::new();
 
     let mut pbs = Vec::new();
 
-    let digits = printable_base10_digits(workers) as usize;
+    let digits = printable_base10_digits(total_chunks) as usize;
 
-    for i in 1..=workers {
+    for _ in 1..=workers {
       let pb = ProgressBar::hidden()
         // no spinner on windows, so we remove the prefix to line up with the progress bar
         .with_style(ProgressStyle::default_spinner().template(if cfg!(windows) {
           "{prefix:.dim} {msg}"
         } else {
           "  {prefix:.dim} {msg}"
         }));
-      pb.set_prefix(format!("[Worker {:>digits$}]", i, digits = digits));
+      pb.set_prefix(format!("[Idle  {}]", " ".repeat(digits)));
```

Comment:
I believe this can be done with a single `format!` like so, avoiding extra allocations: `format!("{:width$}]", "[Idle ", width = digits)`

Reply:
Oops, I commented on the wrong thing.

Reply:
Wait, did you? I think this is a valid improvement

Reply:
It turns out, I'm not sure if we ever actually _show_ this message. But I'll fix it anyway just in case 😅

Reply:
I remember seeing a message similar to `[Idle]` in rav1e-by-gop, but I guess av1an has no time to waste on idle workers 💪